clang/include/clang/Basic/DarwinSDKInfo.h

```diff
@@ -192,6 +192,17 @@ class DarwinSDKInfo {
 Expected<std::optional<DarwinSDKInfo>>
 parseDarwinSDKInfo(llvm::vfs::FileSystem &VFS, StringRef SDKRootPath);
 
+/// Get the system platform prefix for the active target triple.
```

Comment:
I feel this header is used for parsing `SDKSettings.json` only. I don't feel like these functions are in the correct place.

If we can do the cleanup for where my other comment is, we can just limit these function to be local to `Driver/Toolchains/Darwin.cpp`. No other file needs to have this information.